### DISCUSSION TITLE
okunan dosyanın boş olması durumu

### DISCUSSION THREAD

**Discussion by ozkanseyyarer** [gusanmaz/BMB212-Algorithms-2223#5] [Q&A] [answered]
Şayet okunan dosya boş ise çıktı olarak 0.0 vermeli diye not yazmışsınız.
Bu durumda çıktı olarak double değil de int değer döndürerek "0" değerini döndürmesi bir sorun olur mu?

**Comment by gusanmaz** [comment, marked as answer]
Sorun olmaz. Ekran `0` ya da `0.0` ya da `0.0000` gibi degeri sifir olan bir sayi yazdirilabilir.
